CONTRIBUTING.md

```diff
@@ -338,6 +338,19 @@ DOCKER_HUB_REPO=vmware EXTRA_TAG= VERSION_TAG=<version_tag> make all
 
 Update documentation following steps listed below.
```

Comment:
Can you move this line below? Haven't realized while merging @luomiao change for vFile release step